### PR TITLE
Update README with token generation hint

### PR DESCRIPTION
**Pull request by shinriyo**
[for](url) save our time

### CODE REVIEW

**Comment by shinriyo** [comment]
only document.
caused by original

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This action uploads artifacts (.apk or .ipa) to Firebase App Distribution.
 
 ### `token`
 
-**Required** Upload token - see Firebase CLI Reference
+run `firebase login:ci` command and get your token.
```
Comment (wzieba) [inline README.md:16]: Can we please add information without removing old one? Something like this:

```suggestion
**Required** Upload token - see Firebase CLI Reference (tldr; run `firebase login:ci` command to get your token).
```

**Review by wzieba — COMMENTED** [review]
Thanks @shinriyo for this PR, sorry I've missed it.

**Comment by shinriyo** [comment]
@wzieba yes, you can.

**Comment by shinriyo** [comment]
Thank you!

**Pull request merged**